doc/WhatsNew.rst

```diff
@@ -27,6 +27,15 @@ Ver 3.3.0 (unreleased)
 - Changes to 'histogram' and 'stddev' autocuts algorithms:
   - choice of sampling by grid now; useful for mosaics and collages
   - for previous parameter of usecrop=True, use sample=crop
```

Comment:
Fixing this is out of scope, perhaps, but it needs fixing too.
```suggestion

  - choice of sampling by grid now; useful for mosaics and collages
  - for previous parameter of usecrop=True, use sample=crop
```

Reply:
Good catch!